src/inspector_socket_server.cc

```diff
@@ -317,7 +295,37 @@ bool InspectorSocketServer::RespondToGet(InspectorSocket* socket,
   return false;
 }
 
+void InspectorSocketServer::PrintDebuggerReadyMessage() {
+  FILE* out = stderr;
+  std::vector<std::string> ids = delegate_->GetTargetIds();
+
+  fprintf(out,
+    "Debugger listening on port %d.\n"
+    "Warning: This is an experimental feature "
+    "and could change at any time.\n",
+    this->port());
+  if (ids.size() == 1)
+    fprintf(out, "To start debugging, open the following URL in Chrome:\n");
+  else if (ids.size() > 1)
+    fprintf(out, "To start debugging, open the following URLs in Chrome:\n");
+  for (const std::string& id : ids) {
+    fprintf(out,
+            "    chrome-devtools://devtools/bundled/inspector.html?"
+            "experiments=true&v8only=true&ws=%s\n",
+            GetWsUrl(this->host(), this->port(), id).c_str());
+  }
+
+  fprintf(out, "\nFor more info go to %s\n\n", GUIDE_URL);
+
+  fprintf(out, MapsToString(this->GetListResponse()).c_str());
```

Comment:
Maybe just `fprintf(out, "http://%s:%d/json/list", this->host().c_str(), this->port());` instead of printing the whole response?